src/rinq/amqp/internal/amqputil/channel_pool_logging.go

```diff
@@ -0,0 +1,35 @@
+package amqputil
+
+import (
+	"github.com/rinq/rinq-go/src/rinq"
+)
+
+func logChannelPoolStart(
+	logger rinq.Logger,
+) {
+	if !logger.IsDebug() {
+		return
+	}
+
+	logger.Log(
+		"channel pool started",
```

Comment:
It'd be good to log the size here, in much the same way that the other services log their pre-fetch count, etc.

Reply:
Size is not currently stored in the struct data. I guess i can pass the channels slice cap().